care/facility/migrations/0468_alter_facilitypatientstatshistory_unique_together_and_more.py

```
@@ -0,0 +1,41 @@
+            name='facility',
+        ),
+        migrations.RemoveField(
+            model_name='facilityrelatedsummary',
+            name='facility',
+        ),
+        migrations.RemoveField(
+            model_name='localbodyscopedsummary',
+            name='lsg',
+        ),
+        migrations.DeleteModel(
+            name='DistrictScopedSummary',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)